### DISCUSSION TITLE
Specified processes on client

### DISCUSSION THREAD

**Discussion by lamposu** [vpnhood/VpnHood#114] [Q&A] [answered]
Is it possible configuration on the client side, and make only some specified processes use this VPN?

**Comment by bobvhood** [comment, marked as answer]
No. VpnHood uses WinDivert (a wrapper for Windows Filtering Platform) to capture network packets, and unfortunately, there is no trace of processId in that layer. However, it is possible on Android.

**Reply by lamposu** [reply]
Thanks  your quickly feedback. I found this, seems after windiver2.0 has support process ID: https://github.com/basil00/Divert/issues/156

**Reply by bobvhood** [reply]
Please look at this issue: Blacklist/Whitelist by application process on Windows #96 
Yes, we knew it but, VpnHood needs to work in the network later to intercept packets. WFP network later does not support PID.

**Reply by lamposu** [reply]
I got it. Thanks!
